### PR TITLE
new onboarding process for MDE in M365D new portal

### PR DESCRIPTION
**Pull request by malidgi**
the tenant configuration part of this article should be similar to the "Onboard to the service" part on the M365D article- https://docs.microsoft.com/en-us/microsoft-365/security/defender/m365d-enable?view=o365-worldwide#onboard-to-the-service

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/production-deployment.md [hunk 1]
```diff
@@ -77,36 +77,9 @@ To gain access into which licenses are provisioned to your company, and to check
 
 
 ## Tenant Configuration
+Onboarding to Microsoft Defender for Endpoint is simple. From the navigation menu, select any item under the Endpoints section, or and Microsoft 365 Defender feature such as Incidents, Hunting, Action center, or Threat analytics to initiate the onboarding process.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/production-deployment.md]:
```suggestion
Onboarding to Microsoft Defender for Endpoint is easy. From the navigation menu, select any item under the Endpoints section, or any Microsoft 365 Defender feature such as Incidents, Hunting, Action center, or Threat analytics to initiate the onboarding process.
```

File: microsoft-365/security/defender-endpoint/production-deployment.md [hunk 2]
```diff
@@ -77,36 +77,9 @@ To gain access into which licenses are provisioned to your company, and to check
...
-
-    ![Image of geographic location in set up](images/setup-preferences.png)
-
-5. Select **Next**.
-
-     ![Image of final preference set up](images/setup-preferences2.png)
-
-6. Select **Continue**.
-
+From a web browser, navigate to <https://security.microsoft.com>.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/production-deployment.md]:
```suggestion
From a web browser, navigate to the [Microsoft 365 Security Center](https://security.microsoft.com).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**